### PR TITLE
Userinfo

### PR DESCRIPTION
**Pull request by alfmags**

### CODE REVIEW

File: vipps-ecom-api.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 API version: 2.0
 
-Document version 2.2.1.
+Document version 2.2.1.
```
Comment (tht13) [inline vipps-ecom-api.md:5]: bump it, not whitespace

File: vipps-ecom-api.md [hunk 2]
```diff
@@ -59,8 +60,53 @@ See also: [How it works](vipps-ecom-api-howitworks.md).
   - [Reserve](#reserve)
   - [The Vipps landing page](#the-vipps-landing-page)
   - [Capture](#capture)
-  - [Reserve capture](#reserve-capture)
-  - [Direct capture](#direct-capture)
+  - [Reserve capture](#reserve-capture-1)
+  - [Direct capture](#direct-capture-1)
```
Comment (tht13) [inline vipps-ecom-api.md:64]: wtf

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -235,6 +281,39 @@ that Vipps can call to delete the data. The Vipps app allows the user to later
...
 
+## Userinfo
+
+```
+Early draft, this should be considered pilot functionality that we are currently rolling out in our test environemnt
+```
+
+Vipps offers a functionality to ask for a generic conset to access Userinfo. This is based on the [Vipps Login](https://github.com/vippsas/vipps-login-api) solution, but you as a merchant can seemlessly combine the two functionalites in a single user session. Combining both the userinfo and payment elements. This means that a merchant needs both a Vipps Login integration, and a Vipps Ecom integration.
+
+When you initiate a payment add the parameter "Scopes:" to ask for a users consent to share these details. For example like be email, address and name. The scopes are based on [Login's list of socopes](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#scopes).
```
Comment (tht13) [inline vipps-ecom-api.md:292]: user backticks on scopes

Comment (tht13) [reply vipps-ecom-api.md:292]: `scopes`

File: vipps-ecom-api.md [hunk 4]
```diff
@@ -235,6 +281,39 @@ that Vipps can call to delete the data. The Vipps app allows the user to later
...
+
+To request these scopes add the scopes to the initial call to
+[`POST:​/ecomm​/v2​/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST)
+
+The user then consents and pays in the app.
+
+```
+This operation has considered an all or nothing approach, a user must complete a valid payment and consent to all
```
Comment (tht13) [inline vipps-ecom-api.md:300]: why code block? if you want to emphasise use carrots `>` for blockquote

File: vipps-ecom-api.md (hunk 3 above)
Comment (mats93) [inline vipps-ecom-api.md:292]: List of scopes on Login repo needs to be updated to V2 (More scopes and different name on nnin scope)

File: vipps-ecom-api.md (hunk 2 above)
Comment (alfmags) [reply vipps-ecom-api.md:64]: Auto generated by Table of contents :S

File: vipps-ecom-api.md (hunk 1 above)
Comment (alfmags) [reply vipps-ecom-api.md:5]: Fixed

File: vipps-ecom-api.md (hunk 3 above)
Comment (alfmags) [reply vipps-ecom-api.md]: Fixed

File: vipps-ecom-api.md (hunk 4 above)
Comment (alfmags) [reply vipps-ecom-api.md]: Fixed

File: vipps-ecom-api.md (hunk 3 above)
Comment (alfmags) [reply vipps-ecom-api.md]: Agreed, talked to Grutle, changes on the way regarding Login Repo

File: vipps-ecom-api.md (hunk 2 above)
Comment (tht13) [reply vipps-ecom-api.md:64]: This is duplicated please remove

File: vipps-ecom-api.md [hunk 5]
```diff
@@ -235,6 +281,41 @@ that Vipps can call to delete the data. The Vipps app allows the user to later
...
 
+## Userinfo
+
+```
+Early draft, this should be considered pilot functionality that we are currently rolling out in our test environemnt.
+```
+
+Vipps offers a functionality to ask for a generic conset to access Userinfo. This is based on the [Vipps Login](https://github.com/vippsas/vipps-login-api) solution, but you as a merchant can seemlessly combine the two functionalites in a single user session. Combining both the userinfo and payment elements. This means that a merchant needs be registered with both Vipps Login, and Vipps Ecom functionality.
+
+When you initiate a payment add the parameter `Scopes` to ask for a users consent to share these details. For example like be email, address and name. The scopes are based on [Vipps Login's list of socopes](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#scopes).
```
Comment (tht13) [inline vipps-ecom-api.md:292]: are we using a capital in `Scopes` or `scopes`

Comment (alfmags) [reply vipps-ecom-api.md:292]: Non capital, changing

File: vipps-ecom-api.md (hunk 2 above)
Comment (alfmags) [reply vipps-ecom-api.md:64]: changing

Comment (alfmags) [reply vipps-ecom-api.md:64]: Actually not duplicate, distinct sections

Comment (tht13) [reply vipps-ecom-api.md:64]: what about everything added below?

**Comment by tht13** [comment]
> Also you have a referance to netaxept pan hash in the png flow diagram

@alfmags this ☝️

File: vipps-ecom-api.md (hunk 2 above)
Comment (alfmags) [reply vipps-ecom-api.md:64]: Fixed, TOC auto generate shenaningans

File: vipps-ecom-api.md (hunk 5 above)
Comment (tht13) [reply vipps-ecom-api.md:248]: correct

Comment (Larsera) [reply vipps-ecom-api.md:248]: Maybe we also should use the singular form `scope` as the oidc spec (and userinfo after my PR) does? Or maybe it's for the best to use a more explicit naming like `userinfo_scope` or `consent_scope` as `scope` might be confusing in a transaction/payment context.?

https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims
https://tools.ietf.org/html/rfc6749#section-3.3

**Pull request merged**